src/functTest/groovy/com/bmuschko/gradle/docker/tasks/container/DockerLivenessFunctionalTest.groovy

```diff
@@ -66,7 +66,7 @@ class DockerLivenessFunctionalTest extends AbstractGroovyDslFunctionalTest {
                 withCommand(['su', 'postgres', "-c", "/usr/local/bin/pg_ctl stop -m fast"])
                 successOnExitCodes = [0, 1, 137]
                 awaitStatusTimeout = 60000
```

Comment:
Should this use a lower number as well?

Reply:
No. It's the probing we care about that was set to run every 10/30 seconds and unnecessarily dragging the test out.

Reply:
OK